.gitignore

```diff
@@ -7,3 +7,4 @@ Desktop.ini
 node_modules
 **/generated/_icons.sass
 npm-debug.log
+.vscode/
```

Comment:
why does this need to be added?

Reply:
This was added as we use vscode and I’m assuming folks don’t want my settings 😝. 

But having said that it probably shouldn’t be part of the project. I can update the PR 👍

Reply:
does VS code by default add in a directory on projects? if so this may be good to keep in. I know a few folks here use that editor too, so it may be valuable to add to standard git ignore